### ISSUE TITLE
yamllint deletes comments if no YAML entries are in file yet!!!

### ISSUE THREAD

**Issue by rsnodgrass** [adrienverge/yamllint#473]
If you have a blank file that only contains comments 'yamllint' will actually delete all the comments! For example, if you are stubbing out some thoughts on a yaml configuration file, or adding references before putting in yaml.

```yaml
# this is a comment
```

Or:

```yaml
---
# another yaml comment for notes, but no yaml yet
```

Then yamllint actually destroys the file by deleting all the comments, since there is no yaml entry in the file. **It should NOT DELETE all the comments!**

I read over the yamllint documentation and could not find any way to change this behavior. Further, the DEFAULT behavior is such that it destroys users data perhaps without them knowing it yet.

**Comment by andrewimeson** [comment]
@rsnodgrass yamllint doesn't modify yaml files, it only lints them. I cannot reproduce the issue with yamllint v1.26.3

What are you doing that makes you think yamllint is eating the file? Are you sure you aren't using a different tool like yamlfix?

**Comment by rsnodgrass** [comment]
You are completely correct, my mistake! I went back and found yamlfmt was also being called. Sorry for the confusion.
